### PR TITLE
proxy example

### PR DESCRIPTION
**Pull request by DeadNumbers**

### CODE REVIEW

File: proxy/main.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+)
+
+func main() {
+	ctx := context.Background()
+	options := []chromedp.ExecAllocatorOption{
+		chromedp.ProxyServer("socks5://127.0.0.1:9050"),
+	}
+	options = append(options, chromedp.DefaultExecAllocatorOptions[:]...)
+
+	c, cc := chromedp.NewExecAllocator(ctx, options...)
```
Comment (mvdan) [inline proxy/main.go:18]: `actx, ctx := ...`

File: proxy/main.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	defer cc()
+	// create context
```
Comment (mvdan) [inline proxy/main.go:20]: remove this line

File: proxy/main.go [hunk 3]
```diff
@@ -0,0 +1,31 @@
+package main
```
Comment (mvdan) [inline proxy/main.go:1]: add a package doc, like:

```
// Package main shows how to use chromedp via a proxy. In this case, we're connecting via SOCKS5 to a proxy running on port 9050 on the local machine.
```

Comment (ecarter202) [reply proxy/main.go:1]: How would one use a socks5 proxy with auth?

**Pull request closed without merging**